tests/openadapt/privacy/providers/test_private_ai_scrub.py

```diff
@@ -0,0 +1,218 @@
+"""Module to test PrivateAIScrubbingProvider."""
+
+from io import BytesIO
+import base64
+import os
+
+from loguru import logger
+from PIL import Image
+import easyocr
+import requests
+
+from openadapt import config
+from openadapt.privacy.providers.private_ai import PrivateAIScrubbingProvider
+
+scrub = PrivateAIScrubbingProvider()
+
+try:
+    scrub.scrub_text("hello Bob smith")
+except ValueError:
+    import pytest
+
+    pytestmark = pytest.mark.skip(reason="Private AI API key not found or uin")
+
+
+def test_pdf_redaction() -> None:
+    """Test to check that the PDF redaction works."""
+    pdf_path = "tests/assets/sample_llc_1.pdf"
+    redacted_pdf_path = scrub.scrub_pdf(pdf_path)
+
+    url = "https://api.private-ai.com/deid/v3/process/files/base64"
+
+    file_type = "application/pdf"
```

Comment:
`CONTENT_TYPE`

Reply:
Addressed in: 74399ce3d7eb0ae8d20f01cede1a3906a7534703